assets/css/_docs.scss

```diff
@@ -160,7 +160,7 @@ html.docs {
     input {
       border: none;
       border-bottom: 2px solid $defaultLiteColor;
-      font-size: 1.3rem;
+      font-size: 16px;
```

Comment:
Below 1200px, the body scales down to 14px: https://github.com/lodash/lodash.com/blob/master/assets/css/_media-queries.scss#L46-L52